airflow/jobs/scheduler_job.py

```diff
@@ -54,8 +54,9 @@
 from airflow.utils.state import State
 
 
-class DagFileProcessor(AbstractDagFileProcessor, LoggingMixin):
-    """Helps call SchedulerJob.process_file() in a separate process.
+class DagFileProcessorRunner(AbstractDagFileProcessorRunner, LoggingMixin):
```

Comment:
Do we really need a separate class for DagFileProcessorRunner and DagFileProcessor -- the "Runner" version doesn't seem like it adds much (they are always created together in the same process) so I think I'd prefer this as just a single class.

Reply:
I would like to separate classes that are executed in different threads and processes. New process = New class. This will greatly facilitate the understanding of the code. If we combine it, there will be a lot of methods available, but some intended only for execution from the main process level and others from the child process.

Reply:
I think the name DagFileProcessorProcess will be better. This better describes its roles.

Reply:
I like the idea: _**New process = New class**_